packages/trace-viewer/src/ui/networkResourceDetails.css

```diff
@@ -77,6 +76,7 @@
 
 .tab-network .tabbed-pane-tab.selected {
   font-weight: bold;
+  background-color: var(--vscode-list-inactiveSelectionBackground) !important;
```

Comment:
Why is `!important` required? If necessary, the reason should be documented, but I think it's probably not necessary.